docs/resize-rbd.md

```diff
@@ -0,0 +1,283 @@
+# Resize RBD PVCs
+
+For resize to be supported for your kubernetes cluster, the kubernetes version
+running in your cluster should be >= v1.16 and `ExpandCSIVolumes` feature gate
+has to be enabled.
+
+- [Resize RBD PVCs](#resize-rbd-pvcs)
+  - [Filesystem resize on RBD filesystem volume mode PVCs](#filesystem-resize-on-rbd-filesystem-volume-mode-pvcs)
+  - [Resize Block PVC](#resize-block-pvc)
+
+## Filesystem resize on RBD filesystem volume mode PVCs
+
+pvc.yaml
+
+```
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      pv.kubernetes.io/bind-completed: "yes"
+      pv.kubernetes.io/bound-by-controller: "yes"
+      volume.beta.kubernetes.io/storage-provisioner: rbd.csi.ceph.com
+    creationTimestamp: "2019-12-19T05:44:45Z"
+    finalizers:
+    - kubernetes.io/pvc-protection
+    name: rbd-pvc
+    namespace: default
+    resourceVersion: "3557"
+    selfLink: /api/v1/namespaces/default/persistentvolumeclaims/rbd-pvc
+    uid: efe688d6-a420-4041-900e-c5e19fd73ebf
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+    storageClassName: csi-rbd-sc
+    volumeMode: Filesystem
+    volumeName: pvc-efe688d6-a420-4041-900e-c5e19fd73ebf
+  status:
+    accessModes:
+    - ReadWriteOnce
+    capacity:
+      storage: 1Gi
+    phase: Bound
+kind: List
+metadata:
+  resourceVersion: ""
+  selfLink: ""
+```
+
+- Directory size in pod using this PVC
+
+```
+kubectl exec -it csi-rbd-demo-pod sh
+# df -h
```

Comment:
You only need to show the `/var/lib/www/html` directory, so run `df -h /var/lib/www/html` to skip the unneeded (and possibly confusing) output.

Reply:
Done.